app/src/main/java/org/mozilla/fenix/home/recenttabs/view/RecentTabs.kt

```diff
@@ -69,16 +74,25 @@ import org.mozilla.fenix.theme.FirefoxTheme
  * A list of recent tabs to jump back to.
  *
  * @param recentTabs List of [RecentTab] to display.
+ * @param recentSyncedTab The most recent synced tab from another device. Placeholders displayed when null.
  * @param menuItems List of [RecentTabMenuItem] shown long clicking a [RecentTab].
+ * @param shouldDisplaySyncedTab Whether the synced tab card should be displayed.
  * @param onRecentTabClick Invoked when the user clicks on a recent tab.
  * @param onRecentSearchGroupClicked Invoked when the user clicks on a recent search group.
+ * @param onRecentSyncedTabClick Invoked when the user clicks on the recent synced tab.
+ * @param onSyncedTabSeeAllClick Invoked when user clicks on the "See all" button in the synced tab card.
  */
 @Composable
+@Suppress("LongParameterList")
 fun RecentTabs(
     recentTabs: List<RecentTab>,
+    recentSyncedTab: RecentTab.SyncedTab?,
```

Comment:
Can we move this inside of the list of `RecentTab`? That way you can control whether or not to display this SyncedTab by whether or not it is in the list through the `RecentTabsFeature` and we won't need extra parameters `shouldDisplaySyncedTab` or `recentSyncedTab`.

Reply:
Can we punt on this suggestion until the implementation? IMO it could end up that synced tabs make more sense to be separated, since they will won't be observed from the `BrowserStore` and have a lifecycle that's outside of local changes.

Reply:
So my 2 cents here, I think you can actually go ahead with removing `isSyncedTabDisplayed` because you can assume you won't provide a `recentSyncedTab` if the FeatureFlag is false. Since this also just wireframing, I can also imagine you can append the `RecentTab.SyncedTab` to the end of the `recentTabs` for testing purposes as well. 

I understand we might not be observing from the `BrowserStore`, but we will definitely be observing the `HomeFragmentStore`. Now the future consideration I have, would you be considering adding a new state in `HomeFragmentStore` or would you be appending and updating the list in the reducer when you call a dispatch when you handle a synced tab?

I will admit it is slightly unusual for us to land UI code without the implementation.

Reply:
Okay, that's all good food for thought. I'll remove it for now, and switch the synced tab to be part of the list.

Just for posterity, the other consideration I had in adding `isSyncedTabDisplayed` is that it won't only be dictated by the feature flag. It will also be controlled by whether accounts are logged in and have other devices in their sync constellation in the first place. But agreed that might end up being controlled by a different layer anyway, so I was getting a bit ahead of myself